recipes/wasabi/meta.yaml

```
@@ -0,0 +1,42 @@
+{% set name = "wasabi" %}
+{% set version = "0.1.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 1a2ba97e2d960a25cba876806501f2dc68f74470ccef324c0d7aaf82a25a7422 
+build:
+  noarch: python
+  number: 1
```

Build numbers should start from `0` for new versions. It only needs to be bumped when the previous number has been deployed.
```suggestion
  number: 0
```